janusgraph-es/pom.xml

```diff
@@ -11,12 +11,13 @@
     <url>http://janusgraph.org</url>
     <properties>
         <top.level.basedir>${basedir}/..</top.level.basedir>
-        <elasticsearch.test.version>${elasticsearch.version}</elasticsearch.test.version>
-        <!-- Groovy dynamic scripting must be enabled when testing on ES 1.x/2.x servers.
-             From ES 5.0 Painless inline scripts are used which are enabled by default. -->
-        <elasticsearch.test.groovy.inline>false</elasticsearch.test.groovy.inline>
+        <elasticsearch1.test.version>1.7.6</elasticsearch1.test.version>
+        <elasticsearch2.test.version>2.4.6</elasticsearch2.test.version>
+        <elasticsearch5.test.version>5.5.2</elasticsearch5.test.version>
```

Comment:
Do you test with 5.6.0 ?

Reply:
Thanks, I just updated the client and default 5.x test version to 5.6.0. I hadn't seen this version was released.